test/test_constants.py

```diff
@@ -111,15 +114,13 @@ def test_double_array(self) :
 
    def test_dimensions(self) :
       self.assertTrue(len(self.dataset.dimensions) == 1)
-      dimnames = [k for k in self.dataset.dimensions.keys()]
-      self.assertTrue(dimnames[0] == "dim1")
+      self.assertTrue('dim1' in self.dataset.dimensions.keys())
```

Comment:
Yep, use of 'in' is the better idiom. Plus, the keys() method is redundant in both the original and new statements.